src/os/ui/nodetoggle.js

```diff
@@ -99,3 +125,16 @@ os.ui.NodeToggleCtrl.prototype.updateOpacity = function() {
   var children = item.getChildren();
   this.element_.css('opacity', children && children.length > 0 ? '1' : '0');
 };
+
+
+/**
+ * Updates the icons displayed by the directive.
+ *
+ * @private
+ */
+os.ui.NodeToggleCtrl.prototype.updateIcons_ = function() {
+  var toggleIcons = this.scope_['item']['getToggleIcons']();
```

Comment:
I would recommend using a JSDoc `@type` assignment so you can call `.getToggleIcons()` instead of using bracket notation. That will allow the compiler to type check the call. This is done elsewhere in this file with:

```
var item = /** @type {os.ui.slick.SlickTreeNode} */ (this.scope_['item']);
```